### ISSUE TITLE
Model Training

### ISSUE THREAD

**Issue by Diegomoro415** [Diegomoro415/mildew_detection#18]
- Fit the model using the augmented training dataset and validate on the validation set.
- Save the trained model.
